Tests/SonarScanner.MSBuild.Test/AnalysisWarning/NetFrameworkBootstrapperClassTests.cs

```diff
@@ -1,179 +0,0 @@
-﻿/*
```

Comment:
why is this removed?

Reply:
I think we have two options:

- To build and run the tests for multiple target frameworks and one of them to be netFramework48
- Wait with this change 1 month when we update to net framework 4.6.2 as at that point this class will be removed so we can safely remove the test class too.

Reply:
let's wait for 1 month, there's no hurry IMO

Reply:
Ok, I'll address the comments and convert the PR to draft and it can stick around for a month.